src/Host/Client/Impl/Session/RSessionEvaluationCommands.cs

```diff
@@ -77,11 +77,46 @@ public static class RSessionEvaluationCommands {
             return evaluation.EvaluateAsync(script, REvaluationKind.Json);
         }
 
+        public static Task InstallPackage(this IRSessionInteraction interaction, string name) {
+            var script = $"install.packages({name.ToRStringLiteral()})\n";
+            return interaction.RespondAsync(script);
+        }
+        
+        public static Task InstallPackage(this IRSessionInteraction interaction, string name, string libraryPath) {
+            var script = $"install.packages({name.ToRStringLiteral()}, lib={libraryPath.ToRPath().ToRStringLiteral()})\n";
+            return interaction.RespondAsync(script);
+        }
+        
+        public static Task UninstallPackage(this IRSessionInteraction interaction, string name) {
+            var script = $"remove.packages({name.ToRStringLiteral()})\n";
+            return interaction.RespondAsync(script);
+        }
+
+        public static Task UninstallPackage(this IRSessionInteraction interaction, string name, string libraryPath) {
```

Comment:
Maybe just one method with optional libraryPath? It would make code below shorter with fewer ifs